### ISSUE TITLE
Search field does not work on touch based iOS devices

### ISSUE THREAD

**Issue by MauriceS** [RobertoPrevato/jQuery-KingTable#1]
The demo located at http://ugrose.com/content/demos/kingtable/complex-filters.html works fine on desktop browsers, but when I view it with Safari on the iPad, the search field does not seem to work properly.

The first character is displayed, but after that it does not accept any new character.

Other functionality, such as sorting, does not seem to be affected.

**Comment by RobertoPrevato** [comment]
Hi Maurice, thanks for the heads-up, I will dedicate time to this as soon as possible.

**Comment by RobertoPrevato** [comment]
@MauriceS, quite surprisingly, it seems the bug was caused by a CSS rule. I tested using an Apple iPad Mini 16Gb (MD531FD/A), reproducing the same error you described. After removing this CSS rule [`user-select: none`](https://github.com/RobertoPrevato/jQuery-KingTable/commit/1ca2949948a8bd8838eb5dbfae04933a59747502), now the page is working properly on my iPad. Since currently I have only this single device to test, I don't close this issue yet.
